projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1928,17 +1928,19 @@ public function get_api_data( $blog_id ) {
 	/**
 	 * Get the Publicize shares info.
 	 *
+	 * This function is overwritten in class-publicize-wpcom.php
+	 *
 	 * @param string $blog_id The WPCOM blog_id for the current blog.
```

Comment:
Can we fix the type here by changing it to `int` to fix the Phan error?

```suggestion
	 * @param int $blog_id The WPCOM blog_id for the current blog.
```